modules/cli-using-cli.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/getting-started.adoc
+
+[id='cli-using-cli-{context}']
+= Using the CLI
+
+Review the following sections to learn how to complete common tasks using the CLI.
+
+== Creating a project
+
+Use the `oc new-project` command to create a new project.
+
+----
+$ oc new-project my-project
+Now using project "my-project" on server "https://openshift.example.com:6443".
+----
+
+== Creating a new app
+
+Use the `oc new-app` command to create a new app.
+
+----
+$ oc new-app https://github.com/sclorg/cakephp-ex
+--> Found image 40de956 (9 days old) in image stream "openshift/php" under tag "7.2" for "php"
+
+...
+
+    Run 'oc status' to view your app.
```

Comment:
a/app/application everywhere, except for command names :wink:

Reply:
Thanks @soltysh. I changed app to application in my sentence above the example, but left this instance alone since it is output from the CLI.